### DISCUSSION TITLE
What graphics/rendering backend do you need the most using Godot?

### DISCUSSION THREAD

**Discussion by Xrayez** [godotengine/godot-proposals#4396] [Polls]
I realize the current decision (see #877). Just a little poll.

**Comment by Zireael07** [comment]
Haven't voted, because it's difficult to do - most of my projects are still on GLES3. Only my racer game is on 4.x, i.e. Vulkan.

**Reply by Xrayez** [reply]
I'd say the poll is more about what kind of specific features you absolutely need and depend on, not just whether you currently use a rendering backend simply because of its availability. I'm afraid this poll will be biased towards "I want this" rather than "I need this". 😄

**Comment by Mickeon** [comment]
I'm being forgetful about what the Pros and Cons of each one of them are...

**Comment by MangoDragonHub** [comment]
Vulkan and GLES3 are considered the most important, in my opinion. As much as Vulkan has way more rendering capabilities on mobile hardware, there are still modern devices like the Lenovo Chromebook Duet (2021) doesn't have Vulkan support and only supports GLES3

**Comment by rrtt2323** [comment]
Vulkan And GLES3

**Comment by Delpire** [comment]
I use GLES3 unless I am participating in a game jam, then I use GLES2 so that I can support the web export.
